airbyte-integrations/connectors/destination-local-json/src/main/java/io/airbyte/integrations/destination/local_json/LocalJsonDestination.java

```diff
@@ -124,11 +125,16 @@ public StandardNameTransformer getNamingTransformer() {
    * @param config - config object
    * @return absolute path where to write files.
    */
-  private Path getDestinationPath(JsonNode config) {
+  protected Path getDestinationPath(JsonNode config) {
```

Comment:
Any chance we can DRY?

Reply:
The whole destination is a copy/paste... 
Let me create an issue for this: https://github.com/airbytehq/airbyte/issues/1889